config_test.go

```diff
@@ -106,3 +107,39 @@ func TestConfigWithInvalidPaths(t *testing.T) {
 		})
 	}
 }
+
+func TestConfigWithMissingAttributes(t *testing.T) {
+	tests := []struct {
+		desc      string
+		cfg       Config
+		expectErr string
+	}{
+		{
+			desc: "missing_level",
```

Comment:
We don't need to add `_` to subtest case names here. testing.T will do it for us.

```suggestion
			desc: "missing level",
```